### PR TITLE
Sign transaction returns a TransactionPayload

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: bindings/nodejs/lib/client/client.ts [hunk 1]
```diff
@@ -293,7 +293,7 @@ export class Client {
             },
         });
 
-        return parsePayload(JSON.parse(response).payload);
+        return JSON.parse(response).payload;
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/client/client.ts]: Is this correct?

Comment (kwek20) [reply bindings/nodejs/lib/client/client.ts]: No, as TransactionPayload has fields that can have multiple types (Subclasses) it needs to be 
```
const parsed = JSON.parse(response) as Response<TransactionPayload>;
return plainToInstance(TransactionPayload, parsed.payload);
```

Comment (thibault-martinez) [reply bindings/nodejs/lib/client/client.ts]: Ok, was already like that for this type somewhere else so was probably wrong

Comment (thibault-martinez) [reply bindings/nodejs/lib/client/client.ts]: Changed

**Review by DaughterOfMars — DISMISSED** [review]

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
Conflict

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**